Add subfolder for the NCAR node

JuliaKukulies opened the pull request:
This pull requests adds the subfolder for the NCAR node (**hk25-NCARnode** ) where we collect our analysis scripts and example notebooks. This folder also contains subfolders for our seven science topics: 

1. Storm tracking and development of atmospheric convection
2. Cloud microphysics 
3. Land-atmosphere interactions and Hydrology applications 
4. Extreme weather events 
5. Mesoscale circulations 
6. Diurnal cycle and shallow-to-deep convection transition
7. General circulation & upscale effects

andrewgettelman commented:
Greetings. Sub-folder is great. I like the science teams. Observations are good. 

Question: it looks like these notebooks are mostly for native MPAS format. Is that what you expect most people at NCAR to be using? Our intent was mostly to try to work with the common datasets and common formats.

JuliaKukulies commented:
> Greetings. Sub-folder is great. I like the science teams. Observations are good.
> 
> Question: it looks like these notebooks are mostly for native MPAS format. Is that what you expect most people at NCAR to be using? Our intent was mostly to try to work with the common datasets and common formats.

I am working on rewriting the uxarray notebooks with the common format data as input. So you can just wait a little before merging this pull request. Thanks!

JuliaKukulies commented:
I replaced some of the input data for the `uxarray` notebooks with the common HEALPix datasets. Also added an example notebook that shows how to compare the regional and global HEALPix datasets. 

We are still working on finalizing some notebooks here on the NCAR side, but we would appreciate if our subfolder can already be merged now. We will then probably update it one more time on Friday this week. 

Thanks!
Julia

andrewgettelman reviewed (approved):
Fine for now to get things going.

JuliaKukulies commented:
Thanks @andrewgettelman !
